src/app/pages/OnDemandTvPage/VideoPlayer/index.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useContext } from 'react';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { string } from 'prop-types';
+import {
+  CanonicalMediaPlayer,
+  AmpMediaPlayer,
+} from '@bbc/psammead-media-player';
+import pathOr from 'ramda/src/pathOr';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+import getEmbedUrl from '#lib/utilities/getEmbedUrl';
+import getPlaceholderImageUrl from '../../../routes/utils/getPlaceholderImageUrl';
+
+const getMediaInfo = () => ({
+  title: 'On-demand tv',
+  type: 'video',
+});
```

Comment:
Is the casing of this title correct?  Not sure where it appears, but it seems TV at least should be uppercase.

This also probably doesn't need to be a function when it's always the same value
```suggestion
const MEDIA_INFO = {
  title: 'On-demand tv',
  type: 'video',
};
```